File: Anyelo Carrillo php/operaciones.php
<?php
// Solicitar valores al usuario desde la consola o formulario
echo "Ingrese el primer valor: ";
$A = intval(trim(fgets(STDIN))); // Convertir entrada en entero
echo "Ingrese el segundo valor: ";
$B = intval(trim(fgets(STDIN))); // Convertir entrada en entero

// Realizar operaciones matemáticas básicas
$suma = $A + $B;
$resta = $A - $B;
$multi = $A * $B;

// Manejo de división para evitar error si B es 0
$div = ($B != 0) ? ($A / $B) : "No se puede dividir por 0";

// Comparaciones entre los números
$igual = ($A == $B);
$mayor = ($A > $B);
$menor = ($A < $B);

// Mostrar los resultados en pantalla
echo "\n--- Resultados ---\n";
echo "La suma de los números es: $suma\n";
echo "La resta de los números es: $resta\n";
echo "La multiplicación de los números es: $multi\n";
echo "La división de los números es: $div\n";

if ($igual) {
    echo "Ambos números son iguales.\n";
} else {
    echo "El número menor es: " . ($menor ? $A : $B) . "\n";
    echo "El número mayor es: " . ($mayor ? $A : $B) . "\n";
}

File: Anyelo Carrillo Go/Operaciones.go
package main

import "fmt"

func main() {
	// Solicitar al usuario que ingrese los valores
	var A, B int
	fmt.Print("Ingrese valor: ")
	fmt.Scanln(&A)
	fmt.Print("Ingrese valor: ")
	fmt.Scanln(&B)

	// Realizar operaciones aritméticas
	suma := A + B
	fmt.Printf("La suma de los números es: %d\n", suma)

	res := A - B
	fmt.Printf("La resta de los números es: %d\n", res)

	multi := A * B
	fmt.Printf("La multiplicación de los números es: %d\n", multi)

	div := float64(A) / float64(B)
	fmt.Printf("La división de los números es: %.2f\n", div)

	// Comparaciones
	igual := A == B
	fmt.Printf("¿El número %d es igual a %d? %t\n", A, B, igual)

	mayor := A > B
	if mayor {
		fmt.Printf("El número mayor es: %d\n", A)
		fmt.Printf("El número menor es: %d\n", B)
	} else {
		fmt.Printf("El número mayor es: %d\n", B)
		fmt.Printf("El número menor es: %d\n", A)
	}
}
